src/averageAges.js

```diff
@@ -55,7 +81,34 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const findRelative = function(momName, array) {
```

Comment:
as I see this method is used only once, so I suppose it's better to get rid of this function and move body code to the place you called the function